src/docfx/lib/github/UserProfileCache.cs

```diff
@@ -63,9 +66,11 @@ public UserProfile GetByUserEmail(string userEmail)
         /// Create an instance of <see cref="UserProfileCache"/> from local cache
         /// </summary>
         /// <param name="cachePath">the path of the cache file</param>
-        public static UserProfileCache Create(string cachePath)
+        /// <param name="gitToken">the GitHubAccessor to fetch information when missing in cache</param>
+        public static UserProfileCache Create(string cachePath, GitHubAccessor github)
```

Comment:
>github [](start = 79, length = 6)

Do we need to pass in github again? since we already have it from the constructor.

Reply:
This is a static method to help create the instance, it calls constructor internally, which need it.

Reply:
Right, did not notice that

---
In reply to: [208160192](https://github.com/dotnet/docfx/pull/3163#discussion_r208160192) [](ancestors = 208160192)